sites/fast-website/src/app/components/section-header/section-header.template.ts

```diff
@@ -0,0 +1,11 @@
+import { html } from "@microsoft/fast-element";
+import { SectionHeader } from "./section-header";
+
+export const SectionHeaderTemplate = html<SectionHeader>`
+    <template>
+        <slot name="heading"></slot>
+        <p part="short">${x => x.short}</p>
```

Comment:
What is "short"?

Reply:
"short" is the attribute name @bheston had suggested. Another attribute we can also use  "summary" or "sub-header", if short is confusing. It is a new word for me too.

Reply:
I don't see short as being intuitive for this. It doesn't appear to be a sub header either. This seems to be kind-of "badging" the section, though it doesn't map directly to the badge guidance. It appears to be a 1:1 from a visual standpoint from what I can see.

Reply:
I'd call this a `subheading`

Reply:
My primary thought is that this appears to be a direct use of our Badge component. We should consider a slot here if that's the case. It may not be super intuitive since this visually precedes the heading. While that's solvable with CSS, I do wonder if it's intuitive to a user that want's to target that.

Reply:
I think a named slot of "subheading" would be better than an attribute as it would allow the semantic content to remain in the light dom. You can then use the `::slotted` selector to target the slot and provide the styling you want.

Reply:
yeah, I've updated to remove all attributes and will just have name slots and one default slot.